types/stripe/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for stripe 5.0
+// Type definitions for stripe 6.0
```

Comment:
It would probably be better to not bump this, as the changes don't reflect a change between 5.0 and 6.0. That way, people still using 5.x typings will also get this change.

Reply:
I totally agree.

Reply:
No worries; thanks for checking the PR. I've added a commit to revert the version. Let me know if it's ready to squash.